doc/interfaces/gibbs_unringing_flow.rst

```diff
@@ -0,0 +1,72 @@
+.. _gibbs_unringing_flow:
+
+===============
+Gibbs Unringing
+===============
+
+This tutorial walks through the steps to perform Gibbs unringing using DIPY.
+
+You can try this using your own data; we will be using the data in DIPY.
+You can check how to :ref:`fetch the DIPY data<data_fetch>`.
+
+---------------------------
+Suppress Gibbs Oscillations
+---------------------------
+
+Magnetic Resonance (MR) images are reconstructed from the Fourier coefficients
+of acquired k-space images. Since only a finite number of Fourier coefficients
+can be acquired in practice, reconstructed MR images can be corrupted by Gibbs
+artefacts, which is manifested by intensity oscillations adjacent to edges of
+different tissue types [1]_. Although this artefact affects MR images in
+general, in the context of diffusion-weighted imaging, Gibbs oscillations
+can be magnified in derived diffusion-based estimates [1]_, [2]_.
+
+We will use the ``tissue_data`` dataset in DIPY to showcase this denoising
+method. As with any other workflow in DIPY, you can also use your own data!
+
+You may want to create an output directory to save the resulting Gibbs ringing
+-free volume::
+
+    mkdir gibbs_ringing_output
+
+To run the Gibbs unringing method, we need to specify path to the input volumes.
+This path may contain wildcards to process multiple inputs at once. You can
+also specify the optional arguments. In this case, we will be specifying the
+``num_threads`` and ``out_dir``.
+
+So, we will run the command as::
```

Comment:
To run the Gibbs unringing on the data it suffices to execute the ``dipy_gibbs_ringing`` command, e.g.::

Reply:
Reworded in 19f6367